tests/integration/linux_bridge_test.py

```diff
@@ -295,6 +297,19 @@ def test_linux_bridge_over_bond_over_slave_in_one_transaction(bond0):
     assertlib.assert_absent(bridge_name)
 
 
+def test_rollback_for_linux_bridge():
+    original_state = libnmstate.show()
+    bridge_name = TEST_BRIDGE0
+    bridge_state = _create_bridge_subtree_config(("invalid_port",))
+    with pytest.raises(NmstateVerificationError):
+        with linux_bridge(bridge_name, bridge_state) as desired_state:
+            libnmstate.apply(desired_state)
+
+    time.sleep(5)  # Give some time for NetworkManager to rollback
+    current_state = libnmstate.show()
+    assert original_state == current_state
```

Comment:
In the issue, you mentioned there is leftover bridge interface. Please consider to add a check on output of `ip link show <br_name>` to confirm there is no leftover bridge.

Reply:
We are supposed to report *all* interfaces, even if there are no profiles for it. (they will just show up as `DOWN`)
Therefore, using `ip link` should not be needed.

Reply:
That is the issue of the bug we've found AFAIK. The interface is not reported in state after rollback, but it is in kernel. (But not 100% sure here)

https://github.com/nmstate/nmstate/issues/516

Reply:
Where did you see that the interface is not reported? I do not see it in the issue description.

Reply:
I'm not sure actually. nmstate was not able to remove the interface from system, but I don't know whether it was reported in `show` or not.